includes/Core/Util/Tracking.php

```diff
@@ -198,7 +118,8 @@ private function print_standalone_event_tracking_script() {
 	 * @return array Filtered $data.
 	 */
 	private function inline_js_admin_data( $data ) {
```

Comment:
Let's rename this to `inline_js_base_data()` now that it's no longer attached to the admin data filter.